ansible/roles/ovos_hardware_mark2/tasks/vocalfusion.yml

```diff
@@ -5,6 +5,11 @@
     dest: "{{ _ovos_hardware_mark2_vocalfusion_src_path }}"
     version: "{{ ovos_hardware_mark2_vocalfusion_branch }}"
 
+- name: Override variable for newer kernels
+  set_fact:
+    _ovos_hardware_mark2_lib_modules_path: "/lib/modules/{{ ansible_kernel }}"
+  when: ansible_kernel is version('6.6.22', '>=')
```

Comment:
Why note use this variable `_ovos_hardware_mark2_kernel_target` for the `when`?

Reply:
because the variable is set only if {{ ansible_kernel }} is >= 6.6.22, otherwise it is kept with your values from vars/main.yml : 
```
_ovos_hardware_mark2_kernel_target: "{{ '6.6.22-v8+' if 'Raspberry Pi 4' in ovos_installer_raspberrypi else '6.6.22-v8-16k+' }}"
_ovos_hardware_mark2_lib_modules_path: "/lib/modules/{{ _ovos_hardware_mark2_kernel_target }}"
```

Reply:
I mean why not doing this:

```
when: ansible_kernel is version(_ovos_hardware_mark2_kernel_target, '>=')
```